layouts/404.html

```diff
@@ -47,7 +47,7 @@ <h1>{{T "title-404"}}</h1>
           <p>{{T "message-404"}}</p>
           <p></p>
           <p><a href='{{ "" | relLangURL }}'>{{T "Go-to-homepage"}}</a></p>
-          <p><img src='{{ "/images/gopher-404.jpg" | relURL }}' style="width:50%" alt="Page not found!"></p>
+          <p><img src='{{ "/images/prestashop-404.svg" | relURL }}' style="width:50%" alt="Page not found!"></p>
```

Comment:
Is the width 50% fine with preston? Looks like it's really large

Reply:
I like it like this, quite big, but yes we can make it better looking.

I'm a backend dev, I'm not qualified to say whether something looks fine or not ^^

Reply:
If you like it like this, it's fine then, I didn't check it myself

The fact that you're not a front-end dev or a designer doesn't mean that you can't visually validate something don't you think? 🤣

Reply:
I think it's ok to keep to like that for the moment and improve it if needed in the future :)

Reply:
> The fact that you're not a front-end dev or a designer doesn't mean that you can't visually validate something don't you think? 🤣

What I call "fine" or "pretty" is sometimes not approved by frontend developers. I'm too tolerant it seems.

I find bootstrap good-looking for instance XD